src/views/docs-view/docs-view.module.css

```diff
@@ -1,3 +1,22 @@
 .mdxContent {
 	margin-top: 48px;
 }
+
+.contentWithVersions {
+	position: relative;
+
+	& .versionSwitcherWrapper {
+		width: 100%;
+
+		@media (--medium-up) {
```

Comment:
suggestion: let's use a custom media query from those defined [here](https://github.com/hashicorp/dev-portal/blob/brk.feat/a11y-announce-results/src/styles/custom-media.css#L9) to ensure consistent breakpoints!

Reply:
[and done](https://github.com/hashicorp/dev-portal/pull/945/commits/f199daa4cffe74cea11b9e341d590a679d7b9efd#diff-4d5c3e4fe15d4318a0f3ae5ea5bdc0b6cfbf15494fd7ac1a26c63a1cac2fbbdfR11)